docs/getting_started/creating_an_admin.rst

```diff
@@ -13,7 +13,9 @@ For the rest of the tutorial, you'll need some sort of model. In this tutorial,
 
     // src/Entity/BlogPost.php
 
-    // ...
+    /**
+     * ... configure your entity be handled by an ORM
```

Comment:
lets remove them completley, as `ORM` is not really correct, because `ODM` can be used

Reply:
fixed in my last commit